app/scripts/controllers/home.js

```diff
@@ -55,25 +47,49 @@ angular.module('dockstore.ui')
         $scope.tabMode = 'Tool';
       };
 
-      $scope.$watch('searchQueryContainer', function(newValue, oldValue) {
-        $rootScope.searchQueryContainer = newValue;
+      $scope.$watch('searchQuery', function(newValue, oldValue) {
+        $rootScope.searchQuery = newValue;
       });
 
-      $scope.$watch('searchQueryWorkflow', function(newValue, oldValue) {
-        $rootScope.searchQueryWorkflow = newValue;
-      });
+      $scope.random = function() {
```

Comment:
Add a comment here explaining what random is used for and point out that we could replace this with a more relevant metric later